### DISCUSSION TITLE
When I input one letter but got four in visual studio in vue files.

### DISCUSSION THREAD

**Discussion by sotework** [community/community#14830] [Copilot Conversations]
env：
windows11
vs2022 folder mode
vue file in editor

When I input one letter but got four,
like I input ‘a’ got ‘aaaa’, ‘b’ got ‘bbbb’, ‘GetName’ got ‘GGGGeeeettttNNNNaaaammmmeeee’
我输入‘a’得到‘aaaa’，输入‘GetName’得到‘GGGGeeeettttNNNNaaaammmmeeee’
I tap Backspace well delete four letter.
我按Backspace删除的时候也会一次删除四个。

详细信息在这个链接里面，这个问题也是我提出的：
https://developercommunity.visualstudio.com/t/%E5%81%B6%E5%B0%94%E6%88%91%E8%BE%93%E5%85%A51%E4%B8%AA%E5%AD%97%E6%AF%8D%EF%BC%8C%E4%BD%86%E6%98%AF%E6%98%BE%E7%A4%BA%E4%BA%864%E4%B8%AA%EF%BC%8C%E5%83%8F%EF%BC%8CGetName--GGGGeeeet/10008186

**Comment by kzu** [comment]
Looks like a duplicate of https://github.com/github/feedback/discussions/14206

**Comment by kzu** [comment]
I'm unable to reproduce. Suggestions are properly shown, keypresses are proper too:

![vue](https://user-images.githubusercontent.com/169707/165655268-83256047-2121-4fce-a733-c5cd4e623628.gif)
